### PR TITLE
filtering rulewise reports with used=true

### PR DESCRIPTION
**Pull request by happythenewsad**
## WHAT
Filter rulewise report by used=true

## WHY
We don't want unused feedbacks in this particular analysis

### Screenshots
(If applicable. Also, please censor any sensitive data)

### Notion Card Links
https://www.notion.so/quill/Remove-used-false-feedback-from-the-individual-rule-reports-e2e480b082ab4e2f9331e1e8df4ff78d

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? |  yes
Have you deployed to Staging? | not yet
Self-Review: Have you done an initial self-review of the code below on Github? | yes
Design Review: If applicable, have you compared the coded design to the mockups? | (N/A or Yes)

### CODE REVIEW

**Review by emilia-friedberg — APPROVED** [review]

File: services/QuillLMS/spec/lib/rule_feedback_history_spec.rb [hunk 1]
```diff
@@ -107,7 +107,7 @@ def rule_factory(&hash_block)
         # feedback_histories
         f_h1 = create(:feedback_history, rule_uid: so_rule1.uid, entry: "f_h1 lorem")
         f_h2 = create(:feedback_history, rule_uid: so_rule1.uid, entry: "f_h2 ipsum")
-
+
```
Comment (anathomical) [inline services/QuillLMS/spec/lib/rule_feedback_history_spec.rb:110]: Accidental new whitespace?

**Review by anathomical — APPROVED** [review]
Probably worth cleaning up some accidental extra spaces, but this looks good.  Approve of the test updates.

File: services/QuillLMS/spec/lib/rule_feedback_history_spec.rb (hunk 1 above)
Comment (happythenewsad) [reply services/QuillLMS/spec/lib/rule_feedback_history_spec.rb:110]: I added this line to make the groupings more individually consistent.

File: services/QuillLMS/spec/lib/rule_feedback_history_spec.rb [hunk 2]
```diff
@@ -164,7 +164,7 @@ def rule_factory(&hash_block)
       f_h1 = create(:feedback_history, rule_uid: so_rule1.uid, prompt_id: 1)
       f_h2 = create(:feedback_history, rule_uid: so_rule1.uid, prompt_id: 1)
       f_h3 = create(:feedback_history, rule_uid: unused_rule.uid, prompt_id: 1)
-
+
```
Comment (happythenewsad) [inline services/QuillLMS/spec/lib/rule_feedback_history_spec.rb:167]: Same here - it makes sense if you view the raw file

**Pull request merged**